appintheair_exporter.py

```diff
@@ -111,17 +111,28 @@ def main():
         type=str,
         help="File name used for outputted csv. Should include .csv extension"
     )
+    parser.add_argument(
+        '-m', 
+        '--mine',
```

Comment:
What are your thoughts on making this argument `--ownership`? 

I am thinking that then a user can pass nargs of strings (`MINE`, `NOT_MINE`, `UNKNOWN`) and the script will include any matching `trips` if the Ownership matches.